### ISSUE TITLE
I am maintaining a Linux build of Nook Desktop on Flathub

### ISSUE THREAD

**Issue by OpenSauce04** [mn6/nook-desktop#46]
You can find it here: https://flathub.org/apps/camp.nook.nookdesktop

**Comment by Lesik** [comment]
Would be nice to add it as a link on nook.camp :slightly_smiling_face:

**Comment by OpenSauce04** [comment]
> Thanks @OpenSauce04! Would be nice to add it as a link on [nook.camp](https://nook.camp) :slightly_smiling_face: 

I agree, however the author of the project sadly appears to be MIA, so I'm unsure if this will happen

**Comment by Munoida** [comment]
> You can find it here: https://flathub.org/apps/camp.nook.nookdesktop

i wanna thank you so much lmao, i never knew about this until it popped up on flathub not that long ago

**Comment by okseb** [comment]
Is there a reason I can't minimize it anymore?

**Comment by OpenSauce04** [comment]
> Is there a reason I can't minimize it anymore?

You're using Gnome right? You can't minimize it on Gnome because Gnome doesn't have a tray, so the window would just disappear with no way to open it again

Just shove the window into its own workspace, Gnome apps don't typically have a minimize button anyway

**Comment by secretmango** [comment]
would be great to advertise that version too!

**Comment by mikoducky** [comment]
> > Is there a reason I can't minimize it anymore?
> 
> You're using Gnome right? You can't minimize it on Gnome because Gnome doesn't have a tray, so the window would just disappear with no way to open it again
> 
> Just shove the window into its own workspace, Gnome apps don't typically have a minimize button anyway

I'm on Fedora 38, and I know minimize worked last month at some point. There must have been an update to gnome or the app.

**Comment by atrauzzi** [comment]
> > Is there a reason I can't minimize it anymore?
> 
> You're using Gnome right? You can't minimize it on Gnome because Gnome doesn't have a tray, so the window would just disappear with no way to open it again
> 
> Just shove the window into its own workspace, Gnome apps don't typically have a minimize button anyway

While Gnome doesn't ship with a tray, you can enable it.  It would be nice to still have the option to minimize on linux (or enable it as an option).

Another solution might be to try and somehow integrate with other gnome infrastructure like settings, or extensions perhaps?
